### PR TITLE
Go best practices

### PR DESCRIPTION
**Pull request by macleodbroad-wf**
R: @chrislococo-wf 
O: @ralphmarchildon-wf @patkujawa-wf @dustinhiatt-wf

### CODE REVIEW

**Comment by aviary-wf** [comment]
## Raven
Number of Findings: 0

File: docs.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    - title: CSS
+      src: css/conventions.md
+    - title: Golang
+      src: go/style.md
+    - title: HTML
+      src: html/conventions.md
+    - title: JavaScript
+      src: js/README.md
+    - title: Python
+      src: python/style.md
```
Comment (nickmiller-wf) [inline docs.yml]: `style.md` should be `style.rst`

**Comment by nickmiller-wf** [comment]
+1 on docs.yml

File: README.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+The Workiva Styleguide
+======================
+
+This repository is a collection of coding standards implemented by Workiva. They are provided here as a free resource to help other members of the development community, and are always open to suggestions on how they can be improved.
+
+\[A Koan\](<http://thecodelesscode.com/case/94?topic=compromise>)
```
Comment (chrislococo-wf) [inline README.md]: Did you want this to display as `[A Koan](http://thecodelesscode.com/case/94?topic=compromise)` or were you intending this to be a link with the text `A Koan`?

**Comment by macleodbroad-wf** [comment]
@Workiva/devtools-pp

File: README.md (hunk 2 above)
Comment (macleodbroad-wf) [inline README.md]: Stupid RST to MD converter!

**Comment by nickmiller-wf** [comment]
+1 on docs.yml

File: AUTHORS.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+CSS
+===
+
+-   Pat Kujawa &lt;<patkujawa-wf@workiva.com>&gt; [@patkujawa-wf], Primary Author.
```
Comment (chrislococo-wf) [inline AUTHORS.md]: Also, do our github accounts work as emails as well or should these be first.last@workiva.com?

**Comment by chrislococo-wf** [comment]
+1

**Comment by nickmiller-wf** [comment]
Published up on staging at https://dev.inf-dev.workiva.org/#/docs/coding-conventions-(a---z)

There's a problem with our rst conversion I'll have to investigate on Tuesday. But all the other docs seem to be working well

**Pull request merged**